### PR TITLE
Fix bugs and updates (Python-Django and Python-Flask)

### PR DESCRIPTION
**Pull request by ManuelNemirovsky**
# Update List 
* Updating the link to [Rookout+Python](https://docs.rookout.com/docs/rooks-setup.html) guide
* Updating the README file with detailed explanation of instructions.

### CODE REVIEW

File: python-django/README.md [hunk 1]
```diff
@@ -2,25 +2,21 @@
...
+Before following this guide we recommend reading the basic [Python + Rookout](https://docs.rookout.com/docs/rooks-setup.html) guide.
 
 ## Running Django Server with Rookout
-1. *Run the Rookout agent*:
-``` bash
-$ docker run -p 7486:7486 -e "ROOKOUT_TOKEN=<Your-Token>" rookout/agent
-```
 
-2. *Install dependencies*:
+1. *Install dependencies*:
```
Comment (MickaelAlliel) [inline python-django/README.md]: maybe add a step before for cloning the repo - look at other up-to-date examples we have and have the readme be similar

Comment (MickaelAlliel) [reply python-django/README.md]: or merge that step to be Clone and install dependencies

File: python-django/README.md [hunk 2]
```diff
@@ -29,16 +25,3 @@ $ docker run -p 7486:7486 -e "ROOKOUT_TOKEN=<Your-Token>" rookout/agent
 4. *Enjoy the debugging*:
 Go to https://app.rookout.com and start debugging :)
 
-## Rookout Integration explained
-
```
[the remark this answers is not in the document]
Comment (LiranLast) [reply python-django/README.md]: @itielshwartz WDYT?

File: python-django/todolist/settings.py [hunk 3]
```diff
@@ -114,4 +114,5 @@
     },
 ]
 
-from rook import auto_start
+from rook import start
+start()
```
[the remark this answers is not in the document]
Comment (ManuelNemirovsky) [reply python-django/todolist/settings.py]: Done

File: python-django/README.md (hunk 1 above)
Comment (ManuelNemirovsky) [reply python-django/README.md]: Done

Comment (itielshwartz) [reply python-django/README.md]: 👍

File: python-django/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (itielshwartz) [reply python-django/README.md]: copy the snippet for new syntax

**Comment by MickaelAlliel** [comment]
@ManuelNemirovsky Fix conflict and resend review request :) it will be canceled if u commit after i approve

**Pull request merged**